src/org/ods/orchestration/util/Project.groovy

```diff
@@ -610,6 +613,33 @@ class Project {
         return this.getAutomatedTests(componentName, [TestType.UNIT])
     }
 
+    TestResults getAggregatedTestResults() {
+        return aggregatedTestResults == null ? new TestResults() : aggregatedTestResults.deepCopy();
+    }
+
+    void storeAggregatedTestResults(Map testData, Map matchingResult) {
+        if (aggregatedTestResults == null) {
+            aggregatedTestResults = new TestResults()
+        }
+        def testResults = aggregatedTestResults
+        testData.testsuites.each { testSuite ->
+            if (testSuite.errors) {
+                testResults.addError(Integer.parseInt(testSuite.errors))
+            }
+            if (testSuite.skipped) {
+                testResults.addSkipped(Integer.parseInt(testSuite.skipped))
+            }
+            if (testSuite.failures) {
+                testResults.addFailed(Integer.parseInt(testSuite.failures))
+            }
+            if (testSuite.tests) {
+                testResults.addSucceeded(Integer.parseInt(testSuite.tests) -
+                    (testResults.error + testResults.skipped + testResults.failed))
```

Comment:
I think this is wrong, as testResults.* will contain aggregated test results, while testSuite.tests only contains the tests in this concrete suite. I would Parse the different number is local variables, do the calculations and add the results afterwards.

Reply:
Please, can you confirm this is actually a bug?

Reply:
Fixed.